core/components/atoms/_chip/index.tsx

```diff
@@ -21,16 +21,32 @@ export interface GenericChipProps extends BaseProps {
   iconType?: IconType;
   name: Name;
   maxWidth: string | number;
+  size?: TTabSize;
```

Comment:
It's very confusing to use `TTabSize` in the `Chip` component and then passing it to `Text` & `Icon` component. Please create a new type `TChipSize` for this or use some other generic type.